examples/sample-layers/package.json

```diff
@@ -6,14 +6,16 @@
     "babel-polyfill": "^6.16.0",
     "deck.gl": "^4.0.0-beta.2",
     "extrude-polyline": "^1.0.6",
+    "hex2dec": "^1.0.1",
```

Comment:
I generally try to avoid trivial dependencies. They tend to add up.

Is there any reason we can't just use [`parseInt(token, 16)`](http://stackoverflow.com/questions/1981760/parse-hex-ascii-into-numbers). If there is, add a comment to the import or call site.